Cargo.lock

```diff
@@ -709,6 +709,12 @@ version = "0.1.10"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "4785bdd1c96b2a846b2bd7cc02e86b6b3dbf14e7e53446c4f54c92a361040822"
 
+[[package]]
+name = "cfg-if"
+version = "1.0.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd"
+
```

Comment:
There seem to be a number of spurious updates to crates in this commit. Did you run something to add these? Are they needed?

Reply:
I thought that `cargo update` only fetched newly added dependencies locally. Lesson learned! Will revert.